src/memory.js

```diff
@@ -0,0 +1,50 @@
+
+import loadROM from './utils/loadROM.js';
+
+class Memory {
+
+  constructor(rom) {
+
+    if (!rom) {
+      throw new Error('requires ROM');
+    }
+
+    this.data = loadROM(rom);
+  }
+
+  readROM(address) {
+    let romValue;
+    // if we used `if (!value)...` we would get a hit on 0 since it's falsey.
+    if (typeof address != 'number') {
+      romValue = this.data; // returns the whole ROM loaded in
+    }
+    else {
+      romValue = this.data[address];
+    }
+    return romValue;
+  }
+
+  readROMRange(rangeLow, rangeHigh) {
+    // range to read from
+    if (!rangeLow && !rangeHigh) {
+      throw new Error('please provide a range to return');
+    }
+    // smallest number of our range
+    const low = rangeLow;
+    // largest number of our range
+    const high = rangeHigh;
```

Comment:
It looks like `low` is always `rangeLow` and the same for `high` and `rangeHigh`. Why make new variables for them?